src/lib/validators.js

```diff
@@ -83,3 +83,9 @@ export const validateArrayItemsPass: ValidateArrayItemsPass = v => arr =>
   flattenResults(
     arr.map(v).map((res, i) => mapErrors(e => `At item ${i}: ${e}`)(res))
   );
+
+/**
+ * Always an error
+ */
+type AlwaysErr = Errors => Validator;
+export const alwaysErr: AlwaysErr = errs => () => err(errs);
```

Comment:
What do you think about additionally exporting an `alwaysOK` validator? I know we don't have a use-case internally, but from a library author's perspective it seems like a possibly useful function to provide (and would be consistent with `alwaysErr`).